extensions/sql-database-projects/src/dialogs/publishOptionsDialog.ts

```diff
@@ -29,7 +29,7 @@ export class PublishOptionsDialog {
 	}
 
 	protected initializeDialog(): void {
-		this.optionsTab = utils.getAzdataApi()!.window.createTab(constants.publishOptions);
+		this.optionsTab = utils.getAzdataApi()!.window.createTab(constants.publishingOptions);
```

Comment:
You're updating the name here but I don't see any updates in the constants file. Was this intended? If the name already existed do you need to keep the old one around?

Reply:
The constant is already present in the constants file (Line150).